pkgs/development/libraries/qt-5/5.8/qttools/default.nix

```diff
@@ -1,11 +1,28 @@
-{ qtSubmodule, lib, copyPathsToStore, qtbase }:
+{ stdenv, qtSubmodule, makeQtWrapper, copyPathsToStore, qtbase }:
+
+with stdenv.lib;
 
 qtSubmodule {
   name = "qttools";
   qtInputs = [ qtbase ];
-  patches = copyPathsToStore (lib.readPathsFromFile ./. ./series);
+  nativeBuildInputs = [ makeQtWrapper ];
+
+  patches = copyPathsToStore (readPathsFromFile ./. ./series);
   postFixup = ''
     moveToOutput "bin/qdbus" "$out"
     moveToOutput "bin/qtpaths" "$out"
   '';
+
+  postInstall =   ''
+    wrapQtProgram $out/bin/qcollectiongenerator
```

Comment:
Are these all the binaries? If so it's probably better to use a glob here so programs added in a new version are also wrapped.